lms/static/scripts/frontend_apps/components/test/GroupConfigSelector-test.js

```diff
@@ -98,6 +105,14 @@ describe('GroupConfigSelector', () => {
     checkbox.simulate('input');
   }
 
+  async function getOpenSelect(wrapper) {
```

Comment:
They are slightly different.

* One finds a select by simply doing `find('Select`)`, while the other uses a testid to narrow down which specific select to get.
* One loads options synchronously via `find('[role="option"]')`, the other has to wait with `waitForElement(...)` because options are loaded asynchronously.